public/docs/slide-example.md

```diff
@@ -23,31 +28,68 @@ Is the divider of slides
 
 Is the divider of branches
 
+Use the _Space_ key to navigate through all slides.
+
 ----
 
 ### Second branch of first slide
```

Comment:
It should say "of the first slide".

Not only here, also in https://github.com/butlerx/hackmd/blame/50153c65c6ac37ff783b695f5c6bab2ac72ff517/public/docs/slide-example.md#L25